search/includes/classes/class-health.php

```diff
@@ -69,11 +69,20 @@ public function validate_index_entity_count( array $query_args, \ElasticPress\In
 			'entity'   => $indexable->slug,
 			'type'     => ( array_key_exists( 'post_type', $query_args ) ? $query_args['post_type'] : 'N/A' ),
 			'skipped'  => false,
+			'reason'   => 'N/A',
 			'db_total' => 'N/A',
 			'es_total' => 'N/A',
 			'diff' => 'N/A',
 		];
 
+		if ( ! $indexable->index_exists() ) {
+			// If index doesnt exist and we will skip the rest of the check
+			$result['skipped'] = true;
+			$result['reason'] = 'index-not-found';
+			return $result;
+		}
+
+
```

Comment:
Non-blocking nit pick: we should only add one final line break

Reply:
Sounds good
```suggestion
```